lib/query/querybuilder.js

```diff
@@ -816,30 +816,12 @@ class Builder extends EventEmitter {
     return this._union('union all', args);
   }
 
-  // Adds an intersect statement to the query
```

Comment:
why was this whole section deleted?

Reply:
This section was an exact duplication of the `_union` method, so I refactored it to just call `_union` instead of duplicating the code.